modules/reference/pages/command/featureUtility-installFeature.adoc

```diff
@@ -59,12 +59,18 @@ If the version and packaging type aren't specified, the version defaults to the
 |--acceptLicense
 |Accepts the license agreement for the feature if the feature has a different license from the Open Liberty server into which it's installed.
 
+|--featuresBom
+|Specify the Maven coordinate for the features BOM file to install user features. Ensure that the repository also contains the associated features JSON file in the same group ID as the BOM.
```

Comment:
The meaning is vague:
`Specify the Maven coordinate for the features BOM file to install user features.`

 this  could mean either: 

`Use this option to specify the Maven coordinates for the features BOM file that installs the user features.`
or
`To install user features,  use this option to specify the Maven coordinate for the features BOM file.`